libfwupdplugin/fu-self-test.c

```diff
@@ -350,6 +350,26 @@ fu_common_strsafe_func (void)
 	}
 }
 
+static void
+fu_common_uri_scheme_func (void)
+{
+	struct {
+		const gchar *in;
+		const gchar *op;
+	} strs[] = {
+		{ "https://foo.bar/baz",	"https" },
+		{ "ftp://",			"ftp" },
+		{ "ftp:",			"ftp" },
+		{ "foobarbaz",			NULL },
```

Comment:
I think you need a case insensitive test

Reply:
Fixed.